_usage/hint_mapping.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Using Hint Mapping
+nav_order: 9
+---
+
+## Using Query Hint Mapping
+
+Support for query hint mapping is available with Babelfish version 2.3.0. 
+
+Query optimizers do their best to select the best execution plan for your workload based on an engine’s cost model and column/table statistics, but real-world datasets show that query optimizers sometimes don't make the best choices. You can use a query hint to encourage the query optimizer to avoid performance issues caused by bad execution plans.  
+
+A query hint instructs the database engine how to execute a query. Babelfish can use the [pg_hint_plan](https://github.com/ossc-db/pg_hint_plan) extension to create and manage query hints.
+
+
+### Installing pg_hint_plan
+
+Babelfish version 2.3.0 supports version 1.4 of [pg_hint_plan](https://github.com/ossc-db/pg_hint_plan/releases/tag/REL14_1_4_0). To install pg_hint_plan:
+
+1. Download the  `pg_hint_plan14` module. 
+2. Navigate to the top of the source tree. 
+3. Assume the identity of the cluster owner, and run `make`, and then `make install`. 
+4. Modify your PATH environment variable to include the Babelfish PostgreSQL installation.
+
+You are not required to CREATE the pg_hint_plan extension, but you must either LOAD the extension.  Use the following command to load the extension for use in an individual session:
+
+```sql
+postgres=# LOAD 'pg_hint_plan';
+LOAD
+```
+
+Or, to enable the extension globally, add the extension to the end of the `shared_preload_libraries` parameter in the `postgresql.conf` and reload the parameters.
```

Comment:
This is how users can use pg_hint_plan on PG connection. This may be confusing to users, so could you remove this?

Reply:
Gone... thanks!

Reply:
I meant all those lines, i.e., line25-32.

Reply:
I'm looking at the instructions at:  https://github.com/ossc-db/pg_hint_plan#installation
and it looks like the user is still required to load the extension after make/making it.  Can you please check out the Installation section?

Reply:
`LOAD 'pg_hint_plan'` is contained in `EXECUTE sp_babelfish_configure 'enable_pg_hint', 'on'` so users don't need to do that.

Reply:
@2jungkook I've updated the doc to read exactly how the source doc from @suprio-amzn shared it - can you take up this issue with him please?